src/_P036_FrameOLED.ino

```diff
@@ -761,13 +726,18 @@ boolean Plugin_036(uint8_t function, struct EventStruct *event, String& string)
             }
 
             nextFrameToDisplay = LineNo / ScrollingPages.linesPerFrame; // next frame shows the new content
+            if (nextFrameToDisplay == NFrames) {   // corner-case?
+              nextFrameToDisplay--;
```

Comment:
Why not set the next frame to 0?

Reply:
It set the frame to the non-existing 4 when sending text to line 12 and 4 lines/frame, where it should be 3, I'll re-evaluate this, still looking odd to me, but this works...

Reply:
Calculation was insufficiently tested, corrected.